config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -87,6 +87,8 @@
   <!-- there is no way to make line wrap in property value -->
   <suppress id="lineLength"
          files="src[\\/]test[\\/]resources[\\/].*[\\/]customimportorder"/>
+  <suppress id="lineLength"
+         files="src[\\/]test[\\/]resources[\\/].*[\\/]filters[\\/]suppressionxpathfilter[\\/]InputSuppressionXpathFilterEscape.xml"/>
```

Comment:
please remove this suppression and wrap lines in file

Reply:
Ok. I will keep that in mind.

Reply:
Done